app/models/chargeback_rate.rb

```diff
@@ -70,4 +72,9 @@ def self.seed
       end
     end
   end
+
```

Comment:
@tenderlove @chessbyte I have a similar open ticket to disallow deletion of Default Profile, I can implement this new paradigm there as I work on it once this one goes in and we have a stake in the ground.